notebooks/official/custom/custom_training_tensorboard_profiler.ipynb

```diff
@@ -0,0 +1,1639 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # instructions to authenticate your GCP account. This provides access to your
</pre><p>GCP is a deprecated term. Use Google Cloud instead.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/719/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Changed</p>